scripts/app-version.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
```

Comment:
This might be a silly question, but are we calling this script anywhere? On Parsley, we call this script in a few different places, but I can't find any references to this script on Spruce

Reply:
This was actually a great question I was using the wrong app-version script. I just updated it. Nice catch!